core/db/migrate/20170219132415_add_unique_index_to_number_fields.rb

```diff
@@ -0,0 +1,13 @@
+class AddUniqueIndexToNumberFields < ActiveRecord::Migration
```

Comment:
@wuboy0307 @damianlegawiec It's a feature. ;) http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment